### PR TITLE
lambda.part2.exercise1 (Usov)

### PR DESCRIPTION
**Pull request by Sadredneck**

### CODE REVIEW

File: src/test/java/lambda/part2/exercise/Exercise1.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        BiPredicate<Person,Person> sameAgesChecker =
+            (x,y) -> x.getAge() == y.getAge();
+
+        assertTrue(sameAgesChecker.test(person1, person2));
+        assertFalse(sameAgesChecker.test(person1, person3));
+        assertFalse(sameAgesChecker.test(person2, person3));
+    }
+
+    private static String getFullName(Person person) {
+        return person.getFirstName() + person.getLastName();
```
Comment (shaart) [inline src/test/java/lambda/part2/exercise/Exercise1.java]: You forgot 1 space between name and surname

File: src/test/java/lambda/part2/exercise/Exercise1.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import static org.junit.Assert.assertFalse;
+
+@SuppressWarnings({"unused", "ConstantConditions"})
+public class Exercise1 {
+
+    @Test
+    public void ageExtractorFromPersonUsingMethodReference() {
+        Person person = new Person("Иван", "Мельников", 33);
+
+        Function<Person, Integer> ageExtractor = x -> x.getAge();
```
Comment (shaart) [inline src/test/java/lambda/part2/exercise/Exercise1.java]: I'll be glad if you change this expression-lambda to **method reference** ```Person::getAge```

File: src/test/java/lambda/part2/exercise/Exercise1.java (hunk 1 above)
Comment (Sadredneck) [reply src/test/java/lambda/part2/exercise/Exercise1.java]: Added space between first and last names.

File: src/test/java/lambda/part2/exercise/Exercise1.java (hunk 2 above)
Comment (Sadredneck) [reply src/test/java/lambda/part2/exercise/Exercise1.java]: Lambda changed to method reference

**Pull request merged**